setup.py

```diff
@@ -15,14 +15,15 @@
     install_requires=[
         "argcomplete",
         "docutils",
+        "factory_boy",
         "humanize",
         "pyyaml>=5.1",
         "requests",
+        "scikit-learn",
         "sh",
         "tqdm",
-        "factory_boy",
     ],
-    packages=["darwin"],
+    packages=setuptools.find_packages(),
```

Comment:
Without this line, our nested packages (`dataset` and `torch`) won't be installed